websites/M/Minecraft Menu/presence.ts

```diff
@@ -0,0 +1,81 @@
+const presence = new Presence({
+		clientId: "971311477514444800",
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+let title: HTMLElement;
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+			largeImageKey: "logo",
+			startTimestamp: browsingTimestamp,
+		},
+		search = document.querySelector<HTMLInputElement>(
+			"body > div.pusher > div.site-nav.clearfix > div > header > nav > div.right.menu > div > div.ui.right.action.left.icon.input > input"
+		),
+		page = window.location.pathname,
+		[privacy, buttons] = await Promise.all([
+			presence.getSetting<boolean>("privacy"),
+			presence.getSetting<boolean>("buttons"),
+		]);
+	if (privacy) presenceData.details = "Browsing...";
+	else if (search.value) {
+		presenceData.details = "Searching for:";
+		presenceData.state = search.value;
+		presenceData.smallImageKey = "searching";
+	} else if (page === "/") presenceData.details = "Homepage";
+	else if (page.includes("/server")) {
+		title = document.querySelector(
+			"body > div.pusher > section > div.ui.stackable.grid > div.six.wide.column > table > thead > tr > th"
+		);
+		presenceData.details = title.textContent.trim();
+		presenceData.state = document.querySelector(
+			"[class='active item']"
+		).textContent;
+		presenceData.buttons = [
+			{
+				label: "View server",
```

Comment:
Use [Title Case](https://titlecaseconverter.com/rules/#NYT)
```suggestion
				label: "View Server",
```